packages/cfd/src/sass/cfd.scss

```diff
@@ -149,6 +149,7 @@
         &-title {
             margin-top: 2.4rem;
             margin-bottom: 0.8rem;
+            text-align: center;
```

Comment:
If we are using the `Text` component here, can we use `align` prop instead of adding this?